pkg/bindings/connection.go

```diff
@@ -113,13 +113,12 @@ func (c *Connection) DoRequest(httpBody io.Reader, httpMethod, endpoint string,
 		err      error
 		response *http.Response
 	)
-	safePathValues := make([]interface{}, len(pathValues))
+	var safePathValues []interface{}
 	// Make sure path values are http url safe
 	for _, pv := range pathValues {
 		safePathValues = append(safePathValues, url.QueryEscape(pv))
 	}
 	safeEndpoint := fmt.Sprintf(endpoint, safePathValues...)
```

Comment:
Please add a TODO about converting to url library.  Thanks.

Reply:
sure